backend/routes/auth/linkedin.js

```diff
@@ -28,14 +28,34 @@ const getAccessToken = async code => {
     .data;
 };
 
+const extractProfilePicUrl = displayImageObject => {
+  const profilePicArray = displayImageObject.elements;
+  if (profilePicArray && profilePicArray.length) {
+    const largestProfilePic = profilePicArray.pop();
+    return largestProfilePic.identifiers[0].identifier;
+  }
+
+  return null;
+};
+
 const getMemberProfile = async accessToken => {
-  const profileUrl = 'https://api.linkedin.com/v2/me';
+  const profileUrl =
+    'https://api.linkedin.com/v2/me?projection=(id,localizedFirstName,localizedLastName,profilePicture(displayImage~:playableStreams))';
```

Comment:
I am not sure how you did this part? Is 'projection = (id, ...)' a query?

Reply:
Hi @StellarApp, the way I understand it, it seems like projection is a way to specify the fields we want from LinkedIn. Based off [this link](https://developer.linkedin.com/docs/guide/v2/concepts/projections), they said that: ```https://developer.linkedin.com/docs/guide/v2/concepts/projections```